llvm/test/CodeGen/AMDGPU/shufflevector.v3i64.v4i64.ll

```diff
@@ -87,6 +87,7 @@ define void @v_shuffle_v3i64_v4i64__1_u_u(ptr addrspace(1) inreg %ptr) {
 ; GFX942-NEXT:    ;;#ASMSTART
 ; GFX942-NEXT:    ; def v[0:7]
 ; GFX942-NEXT:    ;;#ASMEND
+; GFX942-NEXT:    global_store_dwordx2 v8, v[0:1], s[0:1] offset:16
```

Comment:
Have been looking into this, seems like legalization has opaqued the undef elements so dagcombine can't reach. Working on a patch that propagates build_vector undef elements through bitcast for extract_vector_elt.

Reply:
There are definitely places that do this, I fixed a few of these a few months ago. I'm sure there are more